validated_arch_1/stage1/README.md

```diff
@@ -4,9 +4,50 @@ Install the dependencies for Red Hat OpenStack Services on OpenShift operators
 
 ## Notes
 
-Requires an OpenShift 4.10+ cluster with 3 master/worker combo-nodes, with Metal3 and a provisioning network available
+Requires an OpenShift 4.12+ cluster with 3 master/worker combo-nodes, with Metal3 and a provisioning network available
+
+## Steps
 
 1. Create Namespaces
+```bash
+oc apply -f cert_manager_namespace.yaml -f metallb_namespace.yaml -f nmstate_namespace.yaml
+```
 2. Create OperatorGroups
+```bash
```

Comment:
There multiple things which can be applied in single step from the directories.

Many cases basically just apply the a stage (or stages) in a loop (until dependent CRD definition arrives)  seams to be able to work.

Reply:
Yes, that is true.  The intent in showing things in separate steps was to clearly enumerate them for the user's understanding, but perhaps that is not necessary.

Reply:
Let's decide one way or another in a potential follow-up, if necessary